requirements/edx/base.txt

```diff
@@ -68,7 +68,7 @@ django-js-asset==1.2.2    # via django-mptt
 django-method-override==0.2.0
 django-model-utils==3.0.0
 django-mptt==0.11.0
-django-multi-email-field==0.5.1  # via edx-enterprise
+django-multi-email-field==0.6.1  # via edx-enterprise
```

Comment:
Nice, that release fixes Django 2.2 compatibility.  We were on the verge of needing to fork it and apply a long-open PR for that.